### PR TITLE
[Mac] keyboard download page error

### PR DESCRIPTION
**Pull request by rfaramir**
Replaces PR#1809 which had the wrong base. Darcy cherry picked my changes to mac-kbdl-error2. These fixes are for issue#1808 (client side).
The client side fix is to put just the version number in the API call.

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/152918:id/kvk/index.html)

File: mac/Keyman4MacIM/Keyman4MacIM/InfoPlist.strings [hunk 1]
```diff
@@ -15,6 +15,7 @@ CFBundleDisplayName = "Keyman";
  */
 
 //NSHumanReadableCopyright = "Copyright © 2018 SIL International. All rights reserved.";
-//CFBundleShortVersionString = "Keyman 10.0 for macOS";
+//CFBundleShortVersionString = "10.0";
```
Comment (mcdurdin) [inline mac/Keyman4MacIM/Keyman4MacIM/InfoPlist.strings]: This information needs to come from the `/resources/VERSION.md` file.

Comment (rfaramir) [reply mac/Keyman4MacIM/Keyman4MacIM/InfoPlist.strings]: These are just comments. They *used* to be stuck in here, causing yet another step in every release to change them manually.
I figured by commenting them out, future maintainers wouldn't be tempted to put them back in. I could either take them out entirely or add a comment saying why they are NOT here and where they are: in the User-Defined section of Build Settings.  (and ONLY there, I hope)
There's also a Run Script that checks that the version matches resources/VERSION.md, failing the build if it doesn't match.
I am not confident about hacking a script that would write that value into Apple's xcodeproject file format externally. I figure a script to automatically remind you once per version change was a better compromise.

Comment (mcdurdin) [reply mac/Keyman4MacIM/Keyman4MacIM/InfoPlist.strings]: Okay, perhaps changing these to reflect the place where they need to be updated is better.

I am okay with a test that fails if the version number mismatches, for now.

Comment (rfaramir) [reply mac/Keyman4MacIM/Keyman4MacIM/InfoPlist.strings]: Note added to this file. Pushed.

**Pull request merged**